opencensus/stats/aggregation.py

```diff
@@ -0,0 +1,75 @@
+from opencensus.stats import bucket_boundaries
+from opencensus.stats import aggregation_data
+
+class BaseAggregation(object):
+    def __init__(self, aggregation_type=None, buckets=None):
+        if aggregation_type is not None:
+            self.aggregation = aggregation_type
+
+        if buckets is not None:
+            self.buckets = buckets
+
+    def get_aggregation(self):
+        return self.aggregation
+
+class SumAggregation(BaseAggregation):
+    def __init__(self, sum=None, aggregation_type="sum"):
+        super().__init__(aggregation_type)
+        self.aggregation_type = aggregation_type
+        if sum is not None:
+            self.sum = aggregation_data.SumAggregationDataFloat(sum)
+        else:
+            self.sum = aggregation_data.SumAggregationDataFloat(0)
+
+    def get_aggregation(self):
```

Comment:
In Python we don't really use getters and setters like in Java, because we can get the variable just by calling `self.aggregation_type` in this case. Usually when we use getter method, we add a `@property` decorator before the method signature, and the variable the getter returned is a private variable in this class which has a `_` before the variable name. Then with this decorated getter method, calling `self.aggregation_type` will return the value of the private variable `self._aggregation_type` in this class.

For more details look at this link: https://www.python-course.eu/python3_properties.php